src/BenchmarkDotNet/EventHandlers/CompositeBenchmarkEventHandler.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using BenchmarkDotNet.Reports;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Toolchains.Results;
+using BenchmarkDotNet.Validators;
+
+namespace BenchmarkDotNet.EventHandlers
+{
+    public class CompositeBenchmarkEventHandler : BenchmarkEventHandlerBase
```

Comment:
this type does not need to be `public` 

```suggestion
    internal sealed class CompositeBenchmarkEventHandler : BenchmarkEventHandlerBase
```